apps/settings/js/icc.js

```diff
@@ -285,9 +285,17 @@
             menuItem.identifier);
       var nextActionString = getNextActionString(menu.entries.nextActionList,
         index);
+      // Get the first icon if exists
+      var icon = menuItem.icons && menuItem.icons.length > 0 ?
+        menuItem.icons[0] : null;
```

Comment:
It seems that this function would be used across files, I think you can make another file called `stk_utils` and try to put all of them in it. And also, remember to use requirejs to load it.